Sources/MapboxMaps/Foundation/Projection.swift

```diff
@@ -1,7 +1,10 @@
 import CoreLocation
 
+/// A collection of [Spherical Mercator](http://docs.openlayers.org/library/spherical_mercator.html) projection methods.
```

Comment:
This link is dead; perhaps use this link instead: 
```suggestion
/// A collection of [Spherical Mercator](https://en.wikipedia.org/wiki/Web_Mercator_projection) projection methods.
```